README.md

```diff
@@ -1,2 +1,105 @@
 # terraform-provider
```

Comment:
I would combine this and the line below... something like:

```suggestion
# `terraform-provider` - Ably Terraform Provider
```

Reply:
This is actually my bad - should definitely just be the second line. I copied from aboy-js, and didn't replace this properly, my apologies.

Reply:
Fixed in 38fb4f7c9637b193c15452027b0e9b184d62762a